docs/design/002-airdrop.md

```diff
@@ -0,0 +1,100 @@
+# Design Doc 002: Airdrop Module
+
+## Status
+
+Draft
+
+## Abstract
+
+For the Ojo Network's Airdrop, we intend to require that users are only able to claim the other half of their tokens by staking their entire first half with a validator.
+
+## Context
+
+We want to incentivize airdrop recipients to stake their tokens. An effective way to do this is by rewarding users with the other half of their airdrop allocation if they stake to a validator.
+
+We also need to ensure that we're able to create vesting accounts for the initial airdrop amount, and controlling how long afterwards it will take to unlock. An issue here is that vesting accounts in cosmos can only be created for accounts which do not yet exist.
+
+## Specification
+
+### Msgs
+
+- `CreateAirdropAccount(address, tokensToReceive, amountExpectedToDelegate, vestingLength)` - Create an airdrop account which will unlock into a vesting account, if all the tokens in amountExpectedToDelegate are delegated. This transaction can only occur at genesis.
+- `ClaimAirdrop(fromAddress, toAddress)` - Allows an airdrop recipient to claim the 2nd portion of the airdrop specified in the `CreateAirdropAccount` message.
+  - This transaction will create a new Delayed Vesting Account at `toAddress` with the amount of tokens in `tokensToReceive`. This account will vest as long as `vestingLength` above. This transaction fails if `amountExpectedToDelegate` is not met from the `fromAddress` account. Emits an event once the airdrop has been claimed.
+
+### Parameters
+
+- `ExpiryBlock` - The block at which all unclaimed AirdropAccounts will instead mint tokens into the community pool. After this block, all unclaimed airdrop accounts will no longer be able to be claimed.
+
+### Proposed API
+
+- `QueryAirdropAccount` - Returns an existing airdrop account, along with whether or not it has already been claimed.
```

Comment:
Should this return the status of each half of the airdrop and if they staked the first half?

Reply:
Yeah, and we should also return the account that the 2nd half of the airdrop was deposited into. I'll make this more clear

Reply:
Actually, I think it may be easier for us just to record whether or not it's been "claimed". 
I'm thinking our condition for claiming the other half of the airdrop is by staking a specified amount of Ojo tokens, but that doesn't have to be specific to the initial account itself (e.g., if someone buys more ojo tokens, and stakes that instead, that's fine)

Reply:
Kinda met in the middle with `Returns an existing airdrop account, along with whether or not the user is eligible to claim....`